api/src/main/java/org/pmiops/workbench/config/FileConfigs.java

```diff
@@ -42,6 +42,14 @@ public static JsonNode loadConfig(String path, Class<?> configClass)
     JsonNode marshalledNode = jackson.readTree(marshalledJson);
     JsonNode marshalledDiff = JsonDiff.asJson(configJson, marshalledNode);
 
+    System.out.println("~~~~~~~~~~");
```

Comment:
Can you remove the sysouts or replace them with loggers

Reply:
those are just debugging logs. removing...